src/components/SearchSection.tsx

```diff
@@ -93,74 +93,85 @@ const SearchSection: React.FC<SearchSectionProps> = ({
           onClear={onClear}
         />
       </div>
+      <div className="grid gap-2 grid-cols-[repeat(auto-fit,minmax(12rem,1fr))]">
```

Comment:
NB: @jdhoffa at intermediate viewport widths, this goes onto a second line, with a single filter box on that line. Maybe we can do something to balance that?


<img width="2052" height="466" alt="Image" src="https://github.com/user-attachments/assets/e4bfb8f2-a8bf-4ca7-8115-224a3425bcb8" />